UI/src/view/linear_asset/speedLimitLayer.js

```diff
@@ -507,13 +507,13 @@ window.SpeedLimitLayer = function(params) {
 
     var speedLimits = _.flatten(speedLimitChains);
     drawSpeedLimits(speedLimits, vectorLayer);
-     eventbus.trigger('speedLimits:redrawed', speedLimitChains);
+    eventbus.trigger('speedLimits:redrawed', speedLimitChains);
   };
 
   var drawSpeedLimits = function(speedLimits, layerToUse) {
     var speedLimitsWithType = _.map(speedLimits, function(limit) { return _.merge({}, limit, { type: 'other' }); });
     var speedLimitsWithAdjustments = _.map(speedLimitsWithType, offsetBySideCode);
-    var speedLimitsSplitAt70kmh = _.groupBy(speedLimitsWithAdjustments, function(speedLimit) { return speedLimit.value >= 70; });
+    var speedLimitsSplitAt70kmh = _.groupBy(speedLimitsWithAdjustments, function(speedLimit) { return !_.isUndefined(speedLimit.value) ? speedLimit.value.value >= 70 : false; });
```

Comment:
Can this "value.value" be changed to something more explicit.

Reply:
Let's leave that "value.value" like it is because to switch that we will have a lot of impact on the rest of the code